src/components/pages/Give/GiveForm.tsx

```diff
@@ -228,6 +240,9 @@ class GiveFormView extends React.Component<Props, State> {
               (this.props.apiCallStatus &&
                 this.props.apiCallStatus.status === ApiCallStatusType.STARTED)
             }
+            buttonStyle={{ backgroundColor: "#2196F3" }}
```

Comment:
Doesn't have to happen now but we should gather the disparate colors we use and place them into a palette file like in the web repo, as they're already getting repeated and copy pasted within this PR.

To go into further detail, probably describe them first as things like "accent color, background color, etc..." and then define in the button how they would be mapped to functional uses of those colors (like add a prop to Button that specifies it's nature: "primary button" maps to the accent color, "secondary button" maps to some shade down of the accent color, "danger button" maps to a red thing or something, etc).